src/utils.js

```diff
@@ -436,7 +435,9 @@ export default function createUtilsModule({
     }
 
     return tokens.sort(({ amount: amountA }, { amount: amountB }) => {
-      return web3.utils.toBN(amountA).cmp(web3.utils.toBN(amountB));
+      return ethers.utils
+        .parseEther(amountA)
```

Comment:
they are represented usually in Wei, so parsing it to ether would be incorrect, right?

Reply:
Yep, this seems a mistake i did searching and replacing. Gonna fix that.